src/rpc/net.cpp

```diff
@@ -542,25 +575,6 @@ static RPCHelpMan getnettotals()
     };
 }
 
-static UniValue GetNetworksInfo()
-{
-    UniValue networks(UniValue::VARR);
-    for (int n = 0; n < NET_MAX; ++n) {
-        enum Network network = static_cast<enum Network>(n);
```

Comment:
Any reason to add `enum` in this move-only? The compiler, as well as the dev already know this is an enum

Reply:
Oh, nvm you added `const`, not `enum`.